README.md

```diff
@@ -0,0 +1,41 @@
+## Archives Space End-to-End Tests
+
+#### How to install
+
```

Comment:
add prerequisites -- e.g. Ruby version, OS, etc.

Reply:
I have updated the README file with a link to the ruby version file, as well as more info on installation.
https://github.com/archivesspace/e2e-tests/pull/14/commits/8c2052907c823801e4ab5d282d150be886c08cc5